addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -322,12 +316,85 @@ var addNewCourseTag = function (courseTagName, backend) {
 }];
 };
 
+/**
+ * Adds a new quiz to the given section name.
+ *
+ * @param {string} sectionName
+ * @param {{name: string, questions: {title: string, options: {title: string, isCorrect: boolean}[]}[]}} quizConfig
+ * @param {boolean} backend
+ */
+const addQuizToSection = (sectionName, quizConfig, backend) => {
+    const prefix = backend ? ':iframe ' : '';
+    let quizSteps = [
+    ...addSectionContent(prefix, sectionName),
+{
+    content: 'eLearning: click on quiz',
+    trigger: prefix + 'a[data-slide-category=quiz]',
+    run: "click",
+}, {
+    content: 'eLearning: fill quiz title',
+    trigger: prefix + 'input[name=name]',
+    run: `edit ${quizConfig.name}`,
+}, {
+    content: 'eLearning: fill quiz completion time',
+    trigger: prefix + 'input[name=duration]',
+    run: "edit 10",
+}, {
+    content: 'eLearning: create and publish slide',
+    trigger: prefix + 'footer.modal-footer button:contains("Publish")',
+    run: "click",
+}];
+const addQuestionSteps = quizConfig.questions.flatMap(({ title, options }, questionIdx, questions) => {
+    const createOptionSteps = (option, index) => [
+        {
+            content: `eLearning: enter option ${index + 1}`,
+            trigger: `${prefix} div.list-group .o_wslides_js_quiz_answer:nth-of-type(${index + 1}) input`,
```

Comment:
Minor thing: the space is included in the `prefix` (same on other lines ofc).
```suggestion
            trigger: `${prefix}div.list-group .o_wslides_js_quiz_answer:nth-of-type(${index + 1}) input`,
```